docs/src/current.md

```diff
@@ -0,0 +1,49 @@
+# Current Time
+
+```@setup tz
+using TimeZones
+using Compat.Dates
+```
+
+## `now` / System Time Zone
+
+Julia provides the `now()` method to retrieve your current system's time as a `DateTime`. The TimeZones.jl package provides an additional `now(::TimeZone)` method providing the current time as a `ZonedDateTime`:
+
+```@example tz
+now(tz"Europe/Warsaw")
+nothing; # hide
+```
+
+To get the `TimeZone` currently specified on you system you can use `localzone()`. Combining this method with the new `now` method produces the current system time in the current system's time zone:
+
+```@example tz
+now(localzone())
+nothing; # hide
+```
+
+## `today`
+
+Similar to `now` the TimeZones package also provides a `today(::TimeZone)` method which allows you to determine the current date as a `Date` in the specified `TimeZone`.
+
+```@repl tz
+a, b = now(tz"Pacific/Midway"), now(tz"Pacific/Apia")
+a - b
+today(tz"Pacific/Midway"), today(tz"Pacific/Apia")
+```
+
+You should be careful not to use `today()` when working with `ZonedDateTime`s as you may end up using the wrong day. For example:
+
+```@repl tz
+midway, apia = tz"Pacific/Midway", tz"Pacific/Apia"
+ZonedDateTime(today() + Time(11), midway)
+ZonedDateTime(today() + Time(11), apia)  # Should be 2018-01-30
+ZonedDateTime(today(midway) + Time(11), midway)
+ZonedDateTime(today(apia) + Time(11), apia)
+```
```

Comment:
The output was important here. We need to fake the output here so the comment makes sense.

Reply:
The output is generated for the built product. 

<img width="646" alt="screenshot 2018-10-30 16 54 08" src="https://user-images.githubusercontent.com/3131001/47753121-7bffc480-dc64-11e8-9df4-9d12a95f53b2.png">

Reply:
A `@repl` block generates the output in repl format when the docs are built

Reply:
Can we dynamically update that comment?

Reply:
Originally was:
```
julia> ZonedDateTime(today() + Time(11), apia)  # Should be 2018-01-30
2018-01-29T11:00:00+14:00
```

Reply:
Oh I see, no. I can overwrite `today` for this block so it gives a fixed date

Reply:
Alternatively we can change the comment to: `# Wrong date. With the current rules apia should be one day ahead of midway`

Reply:
I like that better; done